### ISSUE TITLE
EMA of BatchNorm Layer

### ISSUE THREAD

**Issue by liuyuyuil** [CuriousAI/mean-teacher#9]
Hello
I have read the pytorch code of mean-tearcher, and  in the `update_ema_variables` function, only parameters are updated. But as for batchnorm layer, mean and var buffers should also be computed except the weight and bais params.  In the current implementition, batchnorm of EMA model may be using the default 0-mean and 1-var.... am I right ?

**Comment by liuyuyuil** [comment]
No problem. closed.
